kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -162,36 +105,22 @@ ensure_certificates()
     # Required by Azure CLI
     export REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
     echo "REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt" | tee -a /etc/environment > /dev/null
-    
-    if [ $IDENTITY_SYSTEM == "ADFS" ]; then
-        # Trim "adfs" suffix
-        ENDPOINT_ACTIVE_DIRECTORY_ENDPOINT=`echo $METADATA | jq '.authentication.loginEndpoint' | xargs | sed -e 's/adfs*$//' | xargs`
-        log_level -i "ENDPOINT_ACTIVE_DIRECTORY_ENDPOINT: $ENDPOINT_ACTIVE_DIRECTORY_ENDPOINT"
-        
-        CERTIFICATE_PFX_LOCATION="spnauth.pfx"
-        CERTIFICATE_PEM_LOCATION="spnauth.pem"
-        KEY_LOCATION="spnauth.key"
-        CERTIFICATE_LOCATION="spnauth.crt"  
-        
-        convert_secret_to_cert $SPN_CLIENT_SECRET $CERTIFICATE_PFX_LOCATION $CERTIFICATE_PEM_LOCATION $CERTIFICATE_LOCATION $KEY_LOCATION
-        
-        log_level -i "PFX path: '$CERTIFICATE_PFX_LOCATION'"
-        log_level -i "PEM path: '$CERTIFICATE_PEM_LOCATION'"
-        log_level -i "CRT path: '$CERTIFICATE_LOCATION'"
-        log_level -i "KEY path: '$KEY_LOCATION'"
-        
-    fi
 }
 
-# Clone msazurestackworkloads' AKSe fork and move relevant files to the working directory
+# Download msazurestackworkloads' AKSe fork and move relevant files to the working directory
 download_akse()
 {
     # Todo update release branch details: msazurestackworkloads, azsmaster
-    retrycmd_if_failure 5 10 60 git clone https://github.com/msazurestackworkloads/aks-engine -b patch-release-v0.34.2-azs-1904-17 || exit $ERR_AKSE_DOWNLOAD
+    AKSE_REPO="msazurestackworkloads/aks-engine"
+    AKSE_BRANCH="patch-release-v0.36.2-azs"
+
+    AKSE_ZIP_NAME="aks-engine-v0.36.2-linux-amd64"
+    AKSE_ZIP_URL="https://raw.githubusercontent.com/$AKSE_REPO/$AKSE_BRANCH/examples/azurestack/$AKSE_ZIP_NAME.gz"
```

Comment:
Cant we directly download from upstream branch ?.

Reply:
Yeah, I'll make that change. Should we pull the template from the upstream too?